### ISSUE TITLE
Void Map Farming Optimization

### ISSUE THREAD

**Issue by kevrlet** [zininzinin/AutoTrimps#99]
This is working now, however it could be more efficient. The code is not prioritizing mountain or forest maps, meaning you are missing out on a significant amount of resources that would speed up the farming portion.

For example, I usually need about 30 minutes of farming a forest or mountain map to clear my void maps. This ends up boosting my helium/hour from 500 to over 900 before I portal. However, allowing the auto map function to do the void maps for me results in a random map being created; if it lands on food or gems, the farming takes FAR longer to finish. 

tl;dr: In the case of farming for void map completion it's more efficient to prioritize difficulty > type > loot > size for the farming map.

**Comment by spindrjr** [comment]
This should be the case with the latest push. Thanks,
